Lib/test/test_dbm.py

```diff
@@ -75,10 +75,9 @@ def test_anydbm_creation(self):
     def test_anydbm_creation_n_file_exists_with_invalid_contents(self):
         # create an empty file
         test.support.create_empty_file(_fname)
-
         f = dbm.open(_fname, 'n')
```

Comment:
Use maybe: try/finally, just in case.

Reply:
There are 11 other calls of `close()` without try/finally. I had doubts about adding try/finally in all these cases. Maybe in a separate commit.

Reply:
Maybe you can use "with f:" if you don't want to use try/finally :-)

Reply:
It is not supported in 2.7. On other side, this PR will be not applied to 2.7, because test_dbm is rudimentary in 2.7.